CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.0.11
+  - Fixed JRuby resolver bug for versions prior to 9.1.16.0 [#45](https://github.com/logstash-plugins/logstash-filter-dns/pull/45)
```

Comment:
`for JRuby versions prior..`

Reply:
? "Fixed JRuby ... for versions prior" ... not sure how this is confusing?

Reply:
you're right, I was nitpicking